crates/bevy_sprite/src/rect.rs

```diff
@@ -12,6 +12,20 @@ pub struct Rect {
     pub max: Vec2,
 }
 
+/// Struct defining a [`Sprite`](crate::Sprite) border with padding values
+#[repr(C)]
+#[derive(Default, Clone, Copy, Debug, Reflect)]
+pub struct BorderRect {
```

Comment:
This duplicates the `Rect` type, which is re-exported from `taffy`.

Reply:
So should `bevy_sprite` include `taffy` as a dependency?

Reply:
So should bevy_sprite include taffy as a dependency?

Reply:
I'm not sure. I dislike that as it's not a light dependency.

I think we should keep this as it is for now, but probably open an issue once this is merged to think about how we can reduce tech debt here.

Reply:
Now that I think about it I could use a `Rect` directly, with the `min/max` values. Adding a constructor from padding values could do the trick but it sacrifices ergonomics a bit